sums.rb

```diff
@@ -0,0 +1,29 @@
+#a
+class Sum1
+    attr_accessor :total
+
+    def  initialize(var1, var2)
+        @total = var1 + var2
+        @var1 = var1
+        @var2 = var2
+    end
+end
+
+#b
+class Sum2
+
+    def  initialize(a, b)
+        @total = a + b
+        @a = a
+        @b = b
+    end
+    def new_total
+        sum2_total = @a + @b
```

Comment:
We can remove the `sum2_total` and just have `@a + @b`.

![finalclasses](https://user-images.githubusercontent.com/47392623/93383612-bb176e80-f818-11ea-89f0-88de779cdf5b.png)